addons/field_rations/initSettings.inc.sqf

```diff
@@ -101,3 +101,12 @@
     [[-1, 0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8], [LSTRING(Dynamic), "0%", "10%", "20%", "30%", "40%", "50%", "60%", "70%", "80%", 0], 0],
     false
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(zeusUpdates),
+    "CHECKBOX",
+    [LSTRING(ZeusUpdates_DisplayName), LSTRING(ZeusUpdates_Description)],
+    LSTRING(DisplayName),
+    false,
+    true
```

Comment:
```suggestion
    1
```

Reply:
Any reason to use 1 instead of true for `_isGlobal`?

Reply:
Because docs: https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#arguments-of-cba_fnc_addsetting

`true` works, but it's maintained for BWC.